lib/kitchen/driver/dokken.rb

```diff
@@ -429,6 +431,33 @@ def docker_creds
                           end
       end
 
+      def docker_config_creds
+        return @docker_config_creds if @docker_config_creds
+
+        @docker_config_creds = {}
+        config_file = ::File.join(::Dir.home, ".docker", "config.json")
+        JSON.load_file!(config_file)["auths"].each do |k, v|
+          username, password = Base64.decode64(v["auth"]).split(":")
```

Comment:
Hi @evandam 

Whenever the config file has multiple registry entires and one of the registry doesn't have any credentials, this line will raise exception.
```ruby
E, [2022-06-01T00:05:44.812885 #32360] ERROR -- Kitchen: Class: Kitchen::ActionFailed
E, [2022-06-01T00:05:44.812915 #32360] ERROR -- Kitchen: Message: Failed to complete #create action: [undefined method `unpack1' for nil:NilClass

    str.unpack1("m")
       ^^^^^^^^]
E, [2022-06-01T00:05:44.812949 #32360] ERROR -- Kitchen: ----------------------
E, [2022-06-01T00:05:44.812979 #32360] ERROR -- Kitchen: ------Backtrace-------
E, [2022-06-01T00:05:44.813008 #32360] ERROR -- Kitchen: .rvm/rubies/ruby-3.1.0/lib/ruby/3.1.0/base64.rb:59:in `decode64'
E, [2022-06-01T00:05:44.813036 #32360] ERROR -- Kitchen: Projects/kitchen-dokken/lib/kitchen/driver/dokken.rb:443:in `block in docker_config_creds'
E, [2022-06-01T00:05:44.813070 #32360] ERROR -- Kitchen: Projects/kitchen-dokken/lib/kitchen/driver/dokken.rb:442:in `each'
E, [2022-06-01T00:05:44.813099 #32360] ERROR -- Kitchen: Projects/kitchen-dokken/lib/kitchen/driver/dokken.rb:442:in `docker_config_creds'
```
My config file: 
```json
{
   "auths": {
      "https://index.docker.io/v1/": {
         "auth": "BASE64Username&Password"
      },
      "icr.io": {},
      "public.ecr.aws": {}
   }
}
```